### DISCUSSION TITLE
Multiple `source` urls in rewrites

### DISCUSSION THREAD

**Discussion by chinanderm** [vercel/next.js#23363] [Help] [answered]
### Describe the feature you'd like to request

When defining rewrites in `next.config.js`, it would be handy if a `rewrites` object could support an array of `source` paths so that I can rewrite multiple paths to one destination.

### Describe the solution you'd like

Allow the `source` property to accept a string or an array of strings

### Describe alternatives you've considered

The current solution/alternative is to define separate rewrite objects for each source, repeating the destination value in each.

**Comment by timneutkens** [comment, marked as answer]
Rewrites in next.config.js is already a function so you can achieve this with JS:

```js
module.exports = {
  rewrites() {
    return [
        ...['/a', '/b'].map((source) => ({source, destination: '/somewhere-else'}))
    ]
  }
}
```

**Reply by chinanderm** [reply]
A ha! Smart. I'm upset I didn't consider that. Thank you so much 🎉
